Support_Your_Locals/Views/Shared/_Layout.cshtml

```diff
@@ -16,7 +16,7 @@
         }
         else
         {
-            <form method="post">
+            <form method="post" style="display: inline;">
```

Comment:
By default it is good to register out from main, however this time it would have been better to register out from edit because it is not merged right now and this form does not exist in there. It is changed to simple link (because SignOut is changed to Get instead of Post).

Reply:
Changed

Reply:
Not exactly. The main issue is parent branch. I recommended you doing this from edit because of two reasons: first of all edit is newer (it should have been the new main, but since there are some bugs it was not merged) and you as designer should see the current image (because now you are designing what could be non-existant). And secondly, merge conflicts. If you are fond of solving them, then go on, but I really recommend you copying and pasting these commits this to edit. Not much was added, this should not be hard.